submissions/andrewklmn/html-css-cinema-booking/index.html

```diff
@@ -0,0 +1,501 @@
+                  5
+                </label>
+              </article>
+              <article class="seat" role="none">
+                <h1 class="hidden">Seat</h1>
+                <input class="seat_control disappeared" type="checkbox" id="seat_6_6" name="seat_6_6" aria-label="row6, seat6" value="booked" disabled="disabled">
+                <label class="seat_label" aria-hidden="true" for="seat_6_6" title="Taken!">
+                  6
+                </label>
+              </article>
+              <article class="seat" role="none">
+                <h1 class="hidden">Seat</h1>
+                <input class="seat_control disappeared" type="checkbox" id="seat_6_7" name="seat_6_7" aria-label="row6, seat7" value="booked" disabled="disabled">
+                <label class="seat_label" aria-hidden="true" for="seat_6_7" title="Taken!">
+                  7
+                </label>
+              </article>
+              <article class="seat" role="none">
+                <h1 class="hidden">Seat</h1>
+                <input class="seat_control disappeared" type="checkbox" id="seat_6_8" name="seat_6_8" aria-label="row6, seat8" value="booked">
+                <label class="seat_label" aria-hidden="true" for="seat_6_8" title="row #6, seat #8">
+                  8
+                </label>
+              </article>
+              <article class="seat" role="none">
+                <h1 class="hidden">Seat</h1>
+                <input class="seat_control disappeared" type="checkbox" id="seat_6_9" name="seat_6_9" aria-label="row6, seat9" value="booked">
+                <label class="seat_label" aria-hidden="true" for="seat_6_9" title="row #6, seat #9">
+                  9
+                </label>
+              </article>
+              <article class="seat" role="none">
+                <h1 class="hidden">Seat</h1>
+                <input class="seat_control disappeared" type="checkbox" id="seat_6_10" name="seat_6_10" aria-label="row6, seat10" value="booked">
+                <label class="seat_label" aria-hidden="true" for="seat_6_10" title="row #6, seat #10">
+                  10
+                </label>
+              </article>
+              <article class="seat" role="none">
+                <h1 class="hidden">Seat</h1>
+                <input class="seat_control disappeared" type="checkbox" id="seat_6_11" name="seat_6_11" aria-label="row6, seat11" value="booked">
+                <label class="seat_label" aria-hidden="true" for="seat_6_11" title="row #6, seat #11">
+                  11
+                </label>
+              </article>
+            </section>
+          </section>
+          <button class="book_button" type="submit">
+            Buy now
+          </button>
+        </form>
+      </section>
+      <section class="summary_info" aria-label="Ticket view">
+        <h1 class="hidden">Your order:</h1>
+        <ul class="ticket" aria-label="Order details">
```

Comment:
Check all paddings: why they are different?

Reply:
My bad. 
Missed it... didn't check it on a narrow screen.